src/views/certifications/views/landing/index.tsx

```diff
@@ -33,19 +36,13 @@ function CertificationsLandingView({
 					)
 				})}
 			</div>
-			{/* Content Debug */}
-			<pre style={{ border: '1px solid magenta', margin: '0' }}>
-				<code>
-					{JSON.stringify(
-						{
-							faqItems,
-						},
-						null,
-						2
-					)}
-				</code>
-			</pre>
-		</>
+			<div className={s.faqSection}>
+				<CertificationsMaxWidth>
+					<h2 className={s.faqHeading}>{pageContent.faqHeading}</h2>
+					<AccordionWithMdxContent items={faqItems} />
```

Comment:
🎨 I missed this during the accordion review but the designs show the accordion disclosure text content having `color: var(--token-color-foreground-faint)` and it's currently `color: var(--token-color-foreground-primary)`. Hopefully the override isn't too messy for that — feel free to tack that onto VQA follow up work.

<img width="935" alt="Screen Shot 2022-12-14 at 3 45 07 PM" src="https://user-images.githubusercontent.com/36613477/207739472-07a3500a-1785-4c31-84fe-fb6260295370.png">
<img width="917" alt="Screen Shot 2022-12-14 at 3 46 59 PM" src="https://user-images.githubusercontent.com/36613477/207739507-8f419ce9-33c4-4441-9b21-e32949f195a9.png">

Reply:
Ah great catch 💯 Updated this via `MdxP` 👍

Reply:
To clarify here for posterity: `components/accordion` doesn't render anything as content, it accepts content as a render slot. So have made the change here to `accordion-with-mdx-content`.